packages/mrc-ide-covidsim/src/imperial-params.ts

```diff
@@ -132,6 +142,14 @@ export function assignParameters(
   // School closure
   p['Place closure start time'] = 0
   p['Duration of place closure'] = 10000
+
```

Comment:
nit: this comment reads like it's from an email or a github comment (which it is). Would be nice if it were a bit more explanatory. I'll try to come up with something better.